src/rules/order.js

```diff
@@ -244,16 +244,37 @@ function getSorter(ascending) {
   const multiplier = ascending ? 1 : -1;
 
   return function importsSorter(importA, importB) {
-    let result;
-
-    if (importA < importB) {
-      result = -1;
-    } else if (importA > importB) {
-      result = 1;
+    let result = 0;
+
+    if (!importA.includes('/') && !importB.includes('/')) {
+      if (importA < importB) {
+        result = -1;
+      } else if (importA > importB) {
+        result = 1;
+      } else {
+        result = 0;
+      }
```

Comment:
```suggestion
      return importB - importA;
```

Reply:
importB and importA are strings, which would return NaN